readme.md

```diff
@@ -27,76 +27,140 @@ GitHub Enterprise is also supported by [authorizing your own domain in the optio
 - [**Firefox** add-on](https://addons.mozilla.org/en-US/firefox/addon/refined-github-/)
 - Opera - Use [this Opera extension](https://addons.opera.com/en/extensions/details/download-chrome-extension-9/) to install the Chrome version.
 
-
 ## Highlights
 
-- [Mark issues and pull requests as unread](https://cloud.githubusercontent.com/assets/170270/18231475/bdf83e26-72e4-11e6-958f-9ce9431d80eb.png) *(They will reappear in Notifications)*
+<table>
+    <tr>
+        <th>
+            Dashboard cleanup
+        </th>
+    </tr>
+    <tr><!-- no zebra stripes thx --></tr>
+    <tr>
+        <td>
+            <img src="media/screenshot-dashboard.png">
+        </td>
+    </tr>
+</table>
+
+<table>
```

Comment:
I agree HTML tables aren't great, but perhaps we agree on a format that showcases the content better even though its code might not be _super._